src/main/java/net/tazpvp/tazpvp/commands/admin/holograms/HologramCommand.java

```diff
@@ -0,0 +1,185 @@
+package net.tazpvp.tazpvp.commands.admin.holograms;
+
+import net.tazpvp.tazpvp.utils.enums.CC;
+import net.tazpvp.tazpvp.utils.holograms.Hologram;
+import net.tazpvp.tazpvp.utils.holograms.HologramUtil;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Display;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.entity.TextDisplay;
+import org.joml.Vector3f;
+import world.ntdi.nrcore.utils.command.simple.Label;
+import world.ntdi.nrcore.utils.command.simple.NRCommand;
+
+import java.util.List;
+
+public class HologramCommand extends NRCommand {
+
+    public HologramCommand() {
+        super(new Label("hologram", "tazpvp.hologram", "holo"));
+    }
+
+    @Override
+    public boolean execute(CommandSender sender, String[] args) {
+        if (!sender.hasPermission(getLabel().getPermission())) {
+            sendNoPermission(sender);
+            return true;
+        }
+
+        if (!(sender instanceof Player p)) {
```

Comment:
This command can be run by console, no?

Reply:
I don't think so, I used Player#getLocation and I'd rather not have to add location arguments, since it's long enough anyway.